client/src/pages/CreateProduct.js

```diff
@@ -68,28 +69,35 @@ const typeOfProduct = [
   },
 ];
 
-const config = {
-  bucketName: 'harvestcrunch-bakedgoods',
-  region: 'us-east-2',
-  accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY_ID,
-  secretAccessKey: process.env.REACT_APP_AWS_SECRET_KEY,
-  dirName: '5fac90ecf662c13aa4730775',
-}
-
 export default function CreateProduct() {
   const classes = useStyles();
-  const S3Client = new S3(config);
   const [title, setTitle] = useState("");
   const [description, setDescription] = useState("");
   const [amount, setAmount] = useState("");
   const [productType, setProductType] = useState("Cake");
   const [successOpen, setSuccessOpen] = useState(false);
   const [errorOpen, setErrorOpen] = useState(false);
   const [files, setFiles] = useState([]);
+  const authContext = useContext(AuthContext);
+  const { loadUser, user } = authContext;
+  let S3Client = new S3();
+
+  useEffect(() => {
+    loadUser(localStorage.getItem("token"));
```

Comment:
instead of loadingUser here, you may want to move this up the parent, maybe in `App.js` so that it just loads the user once every session